src/EditModePlayerManager.cpp

```diff
@@ -23,6 +23,11 @@ void EditModePlayerManager::AddPlayers(const NoteData& note_data) {
 		player->Init("Player", GAMESTATE->m_pPlayerState[pn], nullptr, nullptr, nullptr, nullptr, nullptr, nullptr, nullptr, nullptr);
 		player.Load(note_data);
 
+		// We set the actor alias here instead of the name to maintain
+		// backward compatability with scripted stepcharts that expect
+		// the Player's actor frame to be unnamed.
+		player->SetAlias("Player" + PlayerNumberToString(pn));
```

Comment:
I think this should be set where the player is actually created. That seems like a more appropriate place to put it instead of in the abstraction here.

Reply:
Woops sorry I misremembered. I forgot we introduced this player manager which replaced the creation of the actual player actor from ScreenEdit to here so this place is fine!